tox.ini

```diff
@@ -8,3 +8,17 @@ extras = test
 sitepackages = False
 commands =
     python {envbindir}/coverage run --source phabfive -p -m pytest -v {posargs}
+
+[testenv:flake8]
+basepython = python3.6
+extras = test
+deps =
+    flake8-assertive
+    flake8-black
+    flake8-bugbear
+    flake8-comprehensions
+    #flake8-docstrings
+    #flake8-import-order
+    flake8-rst-docstrings
```

Comment:
Why not have all of these deps in the dev-requirements or tests_require as well?